monkey/common/request_data/__init__.py

```diff
@@ -0,0 +1 @@
+from .agent_heartbeat import AgentHeartbeat
```

Comment:
We'd have to think what's the best way to re-arrange the common. Right now everything is arbitrarily sorted. Maybe it would be best to sort it by type: hardcoded consts, models, serializers would end up on a separate folders. Regarding your choice to create a `request_data`, I'm not sure what we would put in it. Everything can be request data, no?

Reply:
You're right. Moved this to models.